spinegeneric/cli/generate_figure.py

```diff
@@ -368,6 +374,88 @@ def compute_statistics(df):
     return df, stats
 
 
+def output_text(stats, metric):
```

Comment:
Don't we want to include example of full text somewhere inside script (docstring or help)? Or is not it necessary?

`The intra-site coefficients of variation (COVs) were averaged for each vendor and found to be all just under X%. The inter-site COVs (and inter-site ANOVA p-values) were X% (p=X) for GE, X% (p=X) for Philips and X% (p=X) for Siemens. The inter-vendor difference was significant (p=X), with the Tukey test showing significant differences between VENDOR_X and VENDOR_Y (p-adj=X) and between VENDOR_Y and VENDOR_Z (p-adj=X).`

Reply:
no need for example-- i've updated the docstrings to clarify what this function does